### ISSUE TITLE
Catalog refresh button is broken

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#23356]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
1.  Go to Apps page and click on refresh button

**Result:**  Post gets made but seems like we are getting an error back. 

**Other details that may be helpful:**  Wes looked into it and says it's a backend issue.

**Comment by westlywright** [comment]
@dramich and I discussed this offline. The catalog is smart enough to compare commit hashes and the action `refresh` becomes a noop if the commits are the same. I verified this by adding my fork of the integration-test-catalog as a catalog, adding that catalog and a new branch I created locally as a global catalog in the UI. Then locally I made a change and pushed that to the catalog's repro. From the UI I used the refresh button from the launch page. I verified the logs issues the following line

`Catalog sync done. 1 templates created, 0 templates updated, 0 templates deleted` 

I think we can close this as a non-issue.
